kimchi/src/prover.rs

```diff
@@ -588,9 +588,14 @@ where
                         index_evals.insert(*g, &c[i]);
                     }
                 });
-            if !index.cs.range_check_selector_polys.is_empty() {
-                index_evals.extend(range_check::circuit_gates().iter().enumerate().map(
-                    |(i, gate_type)| (*gate_type, &index.cs.range_check_selector_polys[i].eval8),
+            if index.cs.range_check_selector_polys.is_some() {
+                index_evals.extend(range_check::gadget::circuit_gates().iter().enumerate().map(
+                    |(i, gate_type)| {
+                        (
+                            *gate_type,
+                            &index.cs.range_check_selector_polys.as_ref().unwrap()[i].eval8,
```

Comment:
best practice is to avoid unwrap if you can, for example here you can rewrite this as:

```rust
if let Some(polys) = &index.cs.range_check_selector_polys {
 
 index_evals.extend(range_check::gadget::circuit_gates().iter().enumerate().map(
                    |(i, gate_type)| {
                        (
                            *gate_type,
                            &polys[i].eval8,
```